openapi/p4pa-organization.openapi.yaml

```diff
@@ -226,5 +289,48 @@ components:
             - ORGANIZATION_CONFLICT
             - ORGANIZATION_BAD_REQUEST
             - ORGANIZATION_GENERIC_ERROR
+            - ORGANIZATION_SIL_SERVICE_NOT_FOUND
+            - ORGANIZATION_SIL_SERVICE_CONFLICT
+            - ORGANIZATION_SIL_SERVICE_BAD_REQUEST
+            - ORGANIZATION_SIL_SERVICE_GENERIC_ERROR
```

Comment:
let's remove these error codes, and use application's error codes already existent